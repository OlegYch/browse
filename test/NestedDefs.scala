object NestedDefs
{
	object K {
		object K2
		class K3
		lazy val k4 = {
			def c = "asdf"
			lazy val k5 = "jkl"
			object K6 {
				class K7
			}
			c + k5 + (new K6.K7).hashCode
		}
	}
}