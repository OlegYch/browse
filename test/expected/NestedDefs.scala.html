<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8" ></meta>
        <meta http-equiv="Expires" content="0" />
        <title>NestedDefs.scala</title>
        <script type="text/javascript" src="jquery-all.js"></script>
        <script type="text/javascript" src="linked.js"></script>
        <link rel="stylesheet" type="text/css" href="style.css" title="Style"></link>
    </head>
    <body>
        <pre>
<span class="keyword">object</span> <a title="NestedDefs.type" id="7129">NestedDefs</a>
<a href="#7130" title="NestedDefs.type" class="delimiter">{</a>
	<span class="keyword">object</span> <a title="NestedDefs.K.type" id="16736">K</a> <a href="#16737" title="NestedDefs.K.type" class="delimiter">{</a>
		<span class="keyword">object</span> <a title="NestedDefs.K.K2.type" id="16740">K2</a>
		<span class="keyword">class</span> <a title="class K3 extends AnyRef" id="16742">K3</a>
		<span class="keyword">lazy</span> <span class="keyword">val</span> <a title="String" id="16743">k4</a> = <span class="delimiter">{</span>
			<span class="keyword">def</span> <a title="=&gt; String" id="16746">c</a> = <span title="String(&quot;asdf&quot;)" class="string">&quot;asdf&quot;</span>
			<span class="keyword">lazy</span> <span class="keyword">val</span> <a title="String" id="16747">k5</a> = <span title="String(&quot;jkl&quot;)" class="string">&quot;jkl&quot;</span>
			<span class="keyword">object</span> <a title="K6.type" id="16749">K6</a> <a href="#16750" title="K6.type" class="delimiter">{</a>
				<span class="keyword">class</span> <a title="class K7 extends AnyRef" id="16752">K7</a>
			<a href="#16752" title="AnyRef" class="delimiter">}</a>
			<a href="#16746" title="=&gt; String">c</a> <span title="(x$1: Any)String">+</span> <a href="#16747" title="=&gt; String">k5</a> <span title="(x$1: Any)String">+</span> <span class="delimiter">(</span><span title="K6.K7" class="keyword">new</span> <a href="#16749" title="K6.type">K6</a>.<a href="#16752" title="K6.K7">K7</a><span class="delimiter">)</span>.<span title="()Int">hashCode</span>
		<span class="delimiter">}</span>
	<span class="delimiter">}</span>
<span title="AnyRef" class="delimiter">}</span>
        </pre>
    </body>
</html>
